flasksrc/session.py

```diff
@@ -0,0 +1,145 @@
+from flask_restful import Resource, abort, reqparse
+from flask_apispec import use_kwargs, marshal_with
+from flask_apispec.views import MethodResource
+from flask import request
+from marshmallow import Schema, fields
+
+from flasksrc.db import get_db
+
+
+# Schema definitions
+class ToyNetSessionPostRequestSchema(Schema):
+    toynet_topo_id = fields.Int()
+    toynet_user_id = fields.Str()
+
+class ToyNetSessionPostResponseSchema(Schema):
+    status = fields.Bool()
+    toynet_session_id = fields.Int()
+
+class ToyNetSessionByIdGetRequestSchema(Schema):
+    toynet_session_id = fields.Int()
+
+class ToyNetSessionByIdGetResponseSchema(Schema):
+    topo_id = fields.Int()
+    topology = fields.Str()
+    user_id = fields.Str()
+
+class ToyNetSessionByIdPutRequestSchema(Schema):
+    toynet_session_id = fields.Int()
+
+class ToyNetSessionByIdPutResponseSchema(Schema):
+    pass
```

Comment:
So far I have only defined Schemas where arguments need to be parsed. I'm open to defining outputs, in which case we should create a work item on trello to apply it to the rest of the code base.

Reply:
Resolved from latest push